rq/job.py

```diff
@@ -438,6 +445,7 @@ def to_date(date_str):
         self.result_ttl = int(obj.get('result_ttl')) if obj.get('result_ttl') else None  # noqa
         self._status = as_text(obj.get('status') if obj.get('status') else None)
         self._dependency_id = as_text(obj.get('dependency_id', None))
+        self._dependents_ids = self.dependents_ids
```

Comment:
We shouldn't store `dependent_ids` in Redis? This value won't be accurate since new dependents could be added.